src/Tizen.Multimedia.MediaPlayer/Player/AudioOffload.cs

```diff
@@ -0,0 +1,161 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using static Interop;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// Provides the ability to control the audio offload for <see cref="Multimedia.Player"/>.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class AudioOffload
+    {
+        private IList<MediaFormatAudioMimeType> _supportedFormat;
+
+        /// <summary>
+        /// Gets the <see cref="Multimedia.Player"/> that owns this instance.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public Player Player { get; }
+
+        /// <summary>
+        /// Provides a means to retrieve audio offload information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal AudioOffload(Player player)
+        {
+            Debug.Assert(player != null);
+            Player = player;
+        }
+
+        internal void CheckEnabled()
```

Comment:
It is right?
As a naming, it seem to check enable state, so enabled is normal, others is abnormal.

But, implementation is opposite

Reply:
then i'll change the name to CheckDisabled.